ci-operator/step-registry/idp/htpasswd/idp-htpasswd-ref.yaml

```diff
@@ -7,6 +7,10 @@
   grace_period: 10m
   commands: idp-htpasswd-commands.sh
   cli: latest
+  env:
+  - name: HYPERSHIFT_NAMESPACE
+    default: "clusters"
+    documentation: "The Namespace where to create the HostedCluster and NodePools. Only used in HyperShift CI."
```

Comment:
```suggestion
    documentation: "The namespace where the HostedCluster and NodePools are created. Only used in HyperShift CI."
```

Reply:
I copied directly from the to-be-deleted ci-operator/step-registry/cucushift/hypershift-extended/idp-htpasswd/cucushift-hypershift-extended-idp-htpasswd-ref.yaml which was added by HyperShift QE. So I'd like to keep as they are : )